yarn.lock

```diff
@@ -1555,7 +1555,7 @@
 
 "@mtfh/common@https://github.com/LBHackney-IT/mtfh-frontend-common":
   version "0.0.1"
-  resolved "https://github.com/LBHackney-IT/mtfh-frontend-common#e2292ae5a57eca1ab10790c4420a77aff004d1f6"
+  resolved "https://github.com/LBHackney-IT/mtfh-frontend-common#fa51eacfe2105a590b594edefaf37b2cfdac18e2"

```

Comment:
If we removed the commit hash it would always get the latest from main which would help prevent this issue in the future
```suggestion
  resolved "https://github.com/LBHackney-IT/mtfh-frontend-common

```